content/tutorials/manage-the-rippled-server/configure-peering/enable-link-compression.md

```diff
@@ -0,0 +1,52 @@
+---
+html: enable-link-compression.html
+funnel: Docs
+doc_type: Tutorials
+category: Manage the rippled Server
+subcategory: Configure Peering
+blurb: Save bandwidth by compressing peer-to-peer communications.
+---
+# Enable Link Compression
+
+The `rippled` server can save bandwidth by compressing its [peer-to-peer communications](peer-protocol.html), at a cost of greater CPU usage. If you enable link compression, the server automatically compresses communications with peer servers that also have link compression enabled. The server uses uncompressed communications with peers who do not have have link compression enabled or do not support link compression. [New in: rippled 1.6.0][]
+
+## Steps
+
+To enable link compression on your server, complete the following steps:
+
+### 1. Edit your `rippled`'s config file.
+
```

Comment:
Why not use the following? Seems weird to add the possessive to `rippled`, and using "server" is consistent with the other docs.
  
Edit your `rippled` server's config file.